health/health.go

```diff
@@ -0,0 +1,98 @@
+package health
+
+import (
+	"fmt"
+	"github.com/coreos/pkg/httputil"
```

Comment:
style nit: we usually group the stdlib imports and 3rd party ones separately, separated by a blank line